app/scripts/modules/core/src/pipeline/config/stages/monitorPipeline/MonitorPipelineStageExecutionDetails.tsx

```diff
@@ -58,9 +58,7 @@ export function MonitorPipelineStageExecutionDetails(props: IExecutionDetailsSec
                   }}
                   options={{ inherit: false, reload: 'home.applications.application.pipelines.executionDetails' }}
                 >
-                  <span>
-                    &nbsp;<a target="_self">{status.executionId}</a>
```

Comment:
FYI

> In cases like the following, links are not rewritten; instead, the browser will perform a full page reload to the original link.
>
>Links that contain target element
>Example: <a href="/ext/link?a=b" target="_self">link</a>

from https://docs.angularjs.org/guide/$location#html-link-rewriting

since this isn't angularjs, it doesn't apply, but this is likely why the `_self` was in there.